ros2doctor/ros2doctor/api/__init__.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, name: str):
         self.name = name
         self.items = []
 
-    def add_to_report(self, item_name: str, item_info: str) -> None:
```

Comment:
Nit, I'd recommend keeping the return type annotation `-> None`, unless is there a technical reason it needs to be removed?